packages/ringcentral-integration/modules/ConferenceCall/index.js

```diff
@@ -891,6 +891,51 @@ export default class ConferenceCall extends RcModule {
     return conferenceData;
   }
 
+  /*
```

Comment:
remove this project specific method into our own module which extends then commons's `conferenceCall` module